
{'date': 'Sun Nov 24 2019 13:22:25.938 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 49
  def pegamoeda(self):
  ^
IndentationError: unexpected indent
'''},
{'date': 'Sun Nov 24 2019 13:22:51.659 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 51
  Jogo.moedas -=1
  ^
IndentationError: expected an indented block
'''},
{'date': 'Sun Nov 24 2019 13:24:05.835 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 54
    pegamoeda 
NameError: name 'pegamoeda' is not defined
'''},
{'date': 'Sun Nov 24 2019 13:28:18.846 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 11
  def __init__(self):
  ^
IndentationError: expected an indented block
'''},
{'date': 'Sun Nov 24 2019 13:28:25.8 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 12
  introd = Cena (img = "https://i.imgur.com/Bcnfg0C.png")
  ^
IndentationError: expected an indented block
'''},
{'date': 'Sun Nov 24 2019 13:28:34.257 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 19
  self.travesseiro = Elemento(FOCO, x=70, y=400, w=50, h=50, cena=self.quartotalita, style={"opacity": 0.3}, vai=self._travesseiro)
  ^
IndentationError: unexpected indent
'''},
{'date': 'Sun Nov 24 2019 13:28:38.391 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 21
  def _travesseiro(self, _=0):
  ^
IndentationError: unexpected indent
'''},
{'date': 'Sun Nov 24 2019 13:28:43.943 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 21
  def _travesseiro(self, _=0):
  ^
IndentationError: unexpected indent
'''},
{'date': 'Sun Nov 24 2019 13:28:53.41 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 21
  def _travesseiro(self, _=0):
  ^
IndentationError: unexpected indent
'''},
{'date': 'Sun Nov 24 2019 13:29:00.468 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 25
  colete = Elemento (img = linkcolete,
  ^
IndentationError: unexpected indent
'''},
{'date': 'Sun Nov 24 2019 13:29:06.267 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 9
    class Jogo:
  module <module> line 19
    	self.travesseiro = Elemento(FOCO, x=70, y=400, w=50, h=50, cena=self.quartotalita, style={"opacity": 0.3}, vai=self._travesseiro)
NameError: name 'FOCO' is not defined
'''},
{'date': 'Sun Nov 24 2019 13:29:19.499 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 9
    class Jogo:
  module <module> line 19
    	self.travesseiro = Elemento(FOCO, x=70, y=400, w=50, h=50, cena=self.quartotalita, style={"opacity": 0.3}, vai=self._travesseiro)
AttributeError: 'GUI' object has no attribute 'quartotalita'
'''},
{'date': 'Sun Nov 24 2019 13:31:15.236 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 9
    class Jogo:
  module <module> line 19
    	self.travesseiro = Elemento(FOCO, x=70, y=400, w=50, h=50, cena=self.quartotalita, style={"opacity": 0.3}, vai=self._travesseiro)
AttributeError: 'GUI' object has no attribute 'quartotalita'
'''},
{'date': 'Sun Nov 24 2019 13:37:37.355 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 19
  botao.vai=Quarto()
  ^
IndentationError: unexpected indent
'''},
{'date': 'Sun Nov 24 2019 13:37:45.113 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 19
  botao.vai=Quarto()
  ^
IndentationError: unexpected indent
'''},
{'date': 'Sun Nov 24 2019 13:37:49.505 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 23
  self.quarto = Cena(img = "https://lh5.googleusercontent.com/-fs1hatHWU9s/UUpy0DJqlXI/AAAAAAAAbs4/Vy1LL28sPeY/s400/tumblr_lt4n2aSjsX1qmvaoo.gif")
  ^
IndentationError: expected an indented block
'''},
{'date': 'Sun Nov 24 2019 13:37:54.937 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 27
  Texto(self.quarto, "Não era bem o travesseiro que eu estava procurando...").vai()
  ^
IndentationError: expected an indented block
'''},
{'date': 'Sun Nov 24 2019 13:38:00.310 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
Exception: $locals_$exec_1987.Quarto is not a function
  module '$exec_1987' line 10
class Introducao:
'''},
{'date': 'Sun Nov 24 2019 13:38:10.52 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
Exception: $locals_$exec_1999.Quarto is not a function
  module '$exec_1999' line 10
class Introducao:
'''},
{'date': 'Sun Nov 24 2019 13:40:15.389 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
Exception: $locals_$exec_445.Quarto is not a function
  module '$exec_445' line 10
class Introducao:
'''},
{'date': 'Sun Nov 24 2019 13:41:12.954 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 31
  fc = Quarto()
  ^
IndentationError: unexpected indent
'''},
{'date': 'Sun Nov 24 2019 13:41:20.801 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 31
  fc = Quarto()
  ^
IndentationError: unexpected indent
'''},
{'date': 'Sun Nov 24 2019 13:41:36.841 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 32
    	fc.inicia()
AttributeError: 'Quarto' object has no attribute 'inicia'
'''},
{'date': 'Sun Nov 24 2019 13:41:47.140 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 32
    	fc.inicia()
AttributeError: 'Quarto' object has no attribute 'inicia'
'''},
{'date': 'Sun Nov 24 2019 13:42:26.763 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 31
  fc = Quarto()
  ^
IndentationError: unexpected indent
'''},
{'date': 'Sun Nov 24 2019 13:42:38.353 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 31
  fc = Quarto()
  ^
IndentationError: unexpected indent
'''},
{'date': 'Sun Nov 24 2019 13:42:53.486 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 32
    	fc.inicia()
AttributeError: 'Quarto' object has no attribute 'inicia'
'''},
{'date': 'Sun Nov 24 2019 13:43:06.84 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 32
    	fc.inicia()
AttributeError: 'Quarto' object has no attribute 'inicia'
'''},
{'date': 'Sun Nov 24 2019 13:43:26.10 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 32
    	fc.inicia()
AttributeError: 'Quarto' object has no attribute 'inicia'
'''},
{'date': 'Sun Nov 24 2019 13:43:36.34 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 32
    	fc.inicia()
AttributeError: 'Quarto' object has no attribute 'inicia'
'''},
{'date': 'Sun Nov 24 2019 13:45:06.3 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 32
    	fc.inicia()
AttributeError: 'Quarto' object has no attribute 'inicia'
'''},
{'date': 'Sun Nov 24 2019 13:45:06.611 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 32
    	fc.inicia()
AttributeError: 'Quarto' object has no attribute 'inicia'
'''},
{'date': 'Sun Nov 24 2019 13:45:41.582 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 32
    	fc.inicia()
AttributeError: 'Quarto' object has no attribute 'inicia'
'''},
{'date': 'Sun Nov 24 2019 13:46:35.121 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 22
  fc.inicia()
  ^
IndentationError: unexpected indent
'''},
{'date': 'Sun Nov 24 2019 13:46:49.783 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 22
    	fc.inicia()
AttributeError: 'Quarto' object has no attribute 'inicia'
'''},
{'date': 'Sun Nov 24 2019 13:47:45.158 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 22
  fc.inicia()
  ^
IndentationError: unexpected indent
'''},
{'date': 'Sun Nov 24 2019 13:47:56.997 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 22
    	fc.inicia()
AttributeError: 'Quarto' object has no attribute 'inicia'
'''},
{'date': 'Sun Nov 24 2019 13:51:36.701 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 21
    	Quarto.vai()
AttributeError: 'Quarto' object has no attribute 'vai'
'''},
{'date': 'Sun Nov 24 2019 13:52:40.850 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 20
  def vai(self, _=0):
  ^
IndentationError: unexpected indent
'''},
{'date': 'Sun Nov 24 2019 13:52:56.259 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 20
  def vai(self, _=0):
  ^
IndentationError: unexpected indent
'''},
{'date': 'Sun Nov 24 2019 13:53:08.337 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 21
  self.quartotalita.vai()
  ^
IndentationError: expected an indented block
'''},
{'date': 'Sun Nov 24 2019 13:53:21.493 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 24
    	Quarto.vai()
TypeError: vai missing 1 positional argument: 'self'
'''},
{'date': 'Sun Nov 24 2019 13:53:35.288 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 24
    	self.Quarto.vai()
AttributeError: 'GUI' object has no attribute 'Quarto'
'''},
{'date': 'Sun Nov 24 2019 13:54:22.974 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 24
    	self.quarto.vai()
AttributeError: 'GUI' object has no attribute 'quarto'
'''},
{'date': 'Sun Nov 24 2019 13:56:14.653 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 25
  fc.inicia 
  ^
IndentationError: unexpected indent
'''},
{'date': 'Sun Nov 24 2019 13:56:19.331 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 25
  fc.inicia 
  ^
IndentationError: unexpected indent
'''},
{'date': 'Sun Nov 24 2019 13:57:41.569 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 25
  fc = inicia()
  ^
IndentationError: unexpected indent
'''},
{'date': 'Sun Nov 24 2019 13:57:53.971 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 25
    fc = inicia()
NameError: name 'inicia' is not defined
'''},
{'date': 'Sun Nov 24 2019 13:58:04.713 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 25
    	fc = inicia()
NameError: name 'inicia' is not defined
'''},
{'date': 'Sun Nov 24 2019 13:58:35.665 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 24
    	Quarto.vai()
AttributeError: 'Quarto' object has no attribute 'vai'
'''},
{'date': 'Sun Nov 24 2019 13:58:47.984 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 24
    	Quarto.inicia()
TypeError: inicia() missing 1 positional argument: self
'''},
{'date': 'Sun Nov 24 2019 13:58:59.878 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 24
    	self.quarto.inicia()
AttributeError: 'GUI' object has no attribute 'quarto'
'''},
{'date': 'Sun Nov 24 2019 13:59:10.171 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 24
    	self.Quarto.inicia()
AttributeError: 'GUI' object has no attribute 'Quarto'
'''},
{'date': 'Sun Nov 24 2019 14:01:55.898 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 17
  travesseiro = Elemento(FOCO, x=70, y=400, w=50, h=50, cena=quartotalita, style={"opacity": 0.3}, vai=travesseiro)
  ^
IndentationError: unexpected indent
'''},
{'date': 'Sun Nov 24 2019 14:02:03.880 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 17
  travesseiro = Elemento(FOCO, x=70, y=400, w=50, h=50, cena=quartotalita, style={"opacity": 0.3}, vai=travesseiro)
  ^
IndentationError: unexpected indent
'''},
{'date': 'Sun Nov 24 2019 14:02:08.515 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 19
  travesseiro = Texto (quartotalita, "Não era bem o travesseiro que eu estava procurando.."
                                                                                                           ^
SyntaxError: invalid syntax
'''},